table/options.go

```diff
@@ -368,14 +368,19 @@ func (m Model) WithFilterInputValue(value string) Model {
 // true, the row will be included in the filtered results. If the function
 // is nil, the function won't be used. The filter input is passed as the second
 // argument to the function.
-func (m Model) WithFilterFunc(shouldInclude func(row Row, filterInput string) bool) Model {
+func (m Model) WithFilterFunc(shouldInclude func(columns []Column, row Row, filterInput string) bool) Model {
```

Comment:
Please run all the examples and update as necessary. I think at least 1-2 will have broken from this.